snapcraft/plugins/kernel.py

```diff
@@ -387,6 +387,25 @@ def _parse_kernel_release(self):
                 )
             )
 
+    def _get_kernel_version(self):
+        kernelversion = (
+            subprocess.check_output("make kernelversion", shell=True, cwd=self.builddir)
+            .decode("utf-8")
+            .strip()
+        )
+        if not kernelversion:
+            raise ValueError("make kernelversion didn't produce any version")
+        try:
+            # Linux kernel version is composed of up to 4 elements: version,
+            # patchlevel, sublevel and extraversion - the first three elements
+            # are numbers, while the last one is a an arbitrary ascii
+            # string e.g. 4.18.3-rc6 => '4', '18', '3', 'rc6'
+            self.kernel_version = re.split("\W+", kernelversion)
+            for i in range(0, 3):
+                self.kernel_version[i] = int(self.kernel_version[i])
+        except Exception as e:
+            raise ValueError("Malformed kernel_version: {!r}".format(kernelversion))
```

Comment:
ditto on exception handling. Missing a new sentence on how to fix the issue or excepted format string.